public/index.html

```diff
@@ -28,6 +28,43 @@
     <meta name="theme-color" content="#ffffff">
     <link href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,300;0,400;0,600;0,700;1,300;1,400;1,600;1,700&display=swap" rel="stylesheet">
 
+    <script>
+      const charToSymbol = (c) => {
+        if (typeof c === 'string') c = c.charCodeAt(0)
+
+        if (c >= 'a'.charCodeAt(0) && c <= 'z'.charCodeAt(0)) {
+          return c - 'a'.charCodeAt(0) + 6
+        }
+
+        if (c >= '1'.charCodeAt(0) && c <= '5'.charCodeAt(0)) {
+          return c - '1'.charCodeAt(0) + 1
+        }
+
+        return 0
+      }
+
+      /**
+       * Convert string to create ID in EOS smart contract
+       * Taken from https://github.com/manh-vv/eosjs-name
+       * @param {String} s
+       * @return string to uint64 is not fit to js number, the return value is string format
+       */
+      window.nameToUint64 = (s) => {
+        let n = 0n
+
+        let i = 0
+        for (; i < 12 && s[i]; i++) {
+          n |= BigInt(charToSymbol(s.charCodeAt(i)) & 0x1f) << BigInt(64 - 5 * (i + 1))
+        }
+
+        if (i === 12) {
+          n |= BigInt(charToSymbol(s.charCodeAt(i)) & 0x0f)
+        }
+
+        return n.toString()
+      }
```

Comment:
> Also isn't those functions available at `eosjs` package?

We indeed have this function in `eosjs`! Thanks for the hint I even didn't think that it could be found there... It's much better now!

> Any reasons why it's not on index.js or its own file?

Well, this was a bit of pain for me... I tried to use the algorithm from `eosjs-name` module which uses `BitInt` from the newer JS spec. I had to add a Babel plugin because the build crashed. Then, there were problems with Uglifying the JS because Uglify doesn't support `BigInt`s. Then I decided just [to go raw](https://cambiatus.slack.com/archives/CA83HJAAD/p1604349898405200?thread_ts=1604331058.397900&cid=CA83HJAAD) and added the JS in the header thinking that it's better to fix all those build problems in a separate issue. And now we just don't need it!

Reply:
ok! got it! thanks for the explanation @amiskov